perception/image_projection_based_fusion/launch/roi_cluster_fusion.launch.xml

```diff
@@ -75,6 +75,7 @@
       <param name="use_iou_x" value="false"/>
       <param name="use_iou_y" value="false"/>
       <param name="iou_threshold" value="0.35"/>
+      <param name="unknown_iou_threshold" value="0.1"/>
       <param name="rois_number" value="$(var input/rois_number)"/>
```

Comment:
[Q] `unknown_iou_threshold_` does not need to be added?

Reply:
Did you mean keep it as changable parameter but remove from launch file?